### PR TITLE
update readme - new release ref in usage example

### PR DESCRIPTION
**Pull request by michael-spengler**

### CODE REVIEW

**Comment by asos-craigmorten** [comment]
Ah good spot - would also need to update the values in the "Installation" section as these still point to `2.1.4`.

Looks like the formatted has gone a lil awry - _think_ was using `deno fmt` before to help guide this (but could well be my vscode plugin setup). Would be good to keep the formatting of TS snippets consistent.

**Comment by michael-spengler** [comment]
@asos-craigmorten: 
you're right I'm e.g. always getting rid of semicolons at the end of lines as they come from a time where people wrote several statements in one line (which is dinosaur style imo :) ) 

I'll take better care of consistent formatting next time applying your https://github.com/cmorten/opine/blob/main/.vscode/settings.json settings ... 

P.S. 
opine is awesome - I'm using it for my lectures for the off-chain backend parts --> also my students like it a lot.
https://github.com/michael-spengler/distributed-ledger-technology-hands-on-lecture ...

File: README.md [hunk 1]
```diff
@@ -41,18 +41,20 @@ Fast, minimalist web framework for <a href="https://deno.land/">Deno</a> ported
...
 app.listen(
-  3000,
-  () => console.log("server has started on http://localhost:3000 🚀"),
-);
+    port,
+    () => console.log(`server has started on http://localhost:${port} 🚀`),
+)
 ```
 
 ## Installation
```
Comment (asos-craigmorten) [inline README.md:60]: As discussed version changes needed in the installation section as well.

**Review by asos-craigmorten — CHANGES_REQUESTED** [review]

**Comment by cmorten** [comment]
Superseded by https://github.com/cmorten/opine/pull/167

**Pull request closed without merging**